internal/acctest/oauthtokenexchangegeneratorsettings/oauth_token_exchange_generator_settings_test.go

```diff
@@ -0,0 +1,80 @@
+package acctest_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/provider"
+)
+
+const oauthTokenExchangeGeneratorSettingsId = "exampleGeneratorGroup"
+
+// Attributes to test with. Add optional properties to test here if desired.
+type oauthTokenExchangeGeneratorSettingsResourceModel struct {
+	defaultGeneratorGroupRefId string
+}
+
+func TestAccOauthTokenExchangeGeneratorSettings(t *testing.T) {
```

Comment:
Would it be possible to add two generator groups to the server profile and then have this test swap between them, so that we cover update here?

Reply:
yes, this is now in place